kDrive/UI/Controller/Files/Save File/SaveFileViewController.swift

```diff
@@ -181,6 +181,19 @@ class SaveFileViewController: UIViewController {
         }
     }
 
+    private func dismiss(animated: Bool, clean: Bool = true, completion: (() -> Void)? = nil) {
+        Task {
+            // Cleanup file that were duplicated to appGroup on extension mode
```

Comment:
We could abstract where the file is stored and if it needs some form of duplication and cleanup, but I won't have the time to do it with this PR